docs/programming/sprites.md

```diff
@@ -49,8 +49,8 @@ The manager works in block, as well as raw copperlist mode.
 void myGsCreate(void) {
   // Some stuff goes here...
 
-  spriteManagerCreate(s_pView, 0); // For raw copper mode, pass position on
-                                   // copperlist for sprite initialization.
+  spriteManagerCreate(s_pView, 0, NULL); // For raw copper mode, pass position on
```

Comment:
we don't use NULL too much in ACE (usually zero) but now I think we should do that more often

Reply:
Yeah, I like `NULL` way more just because it communicates to me I'm passing this for a pointer, not an integral